.travis.yml

```diff
@@ -1,13 +1,12 @@
-sudo: false
 language: node_js
-cache:
-  directories:
-    - node_modules
+cache: npm
```

Comment:
Travis has smart npm caching which adapts to the commands that are used in the pipeline; it appears it is now the default so the cache declaration could even be removed without any downside. 

https://docs.travis-ci.com/user/caching/#npm-cache